### PR TITLE
fix reference to controlManagerExtraAargs

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
According to @neolit123 suggestion, I fix the bad reference to controlManagerExtraAargs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes # https://github.com/kubernetes/kubeadm/issues/1178

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by yuexiao-wang** [comment]
@neolit123 PTAL

**Comment by neolit123** [comment]
@yuexiao-wang 
can you also submit a PR for the `release-1.12` branch fixing the typo in the v1alpha3 doc.go, please?

/lgtm
/ok-to-test
/kind bug
/assign @fabriziopandini

**Comment by yuexiao-wang** [comment]
Sure, I will send a PR.

**Comment by neolit123** [comment]
forgot that we don't need to cherry pick fixes related to godocs.
this PR is sufficient.

File: cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ limitations under the License.
 // 	controlPlaneEndpoint: "10.100.0.1:6443"
 // 	apiServerExtraArgs:
 // 	  authorization-mode: "Node,RBAC"
-// 	controlManagerExtraArgs:
+// 	controllerManagerExtraArgs :
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go]: there is an extra space before `:`
[1]

File: cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go [hunk 2]
```diff
@@ -202,7 +202,7 @@ limitations under the License.
 // 	controlPlaneEndpoint: "10.100.0.1:6443"
 // 	apiServerExtraArgs:
 // 	  authorization-mode: "Node,RBAC"
-// 	controlManagerExtraArgs:
+// 	controllerManagerExtraArgs :
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go]: [1]

**Comment by yuexiao-wang** [comment]
remove some redundant blanks. PTAL

File: cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go (hunk 2 above)
Comment (yuexiao-wang) [reply cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go]: Updated. thanks

Comment (neolit123) [reply cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go]: hm the space before ":" seems to be here still.

**Comment by yuexiao-wang** [comment]
Done

**Comment by neolit123** [comment]
thank, you!
will now wait for an approver to merge it

**Review by fabriziopandini — COMMENTED** [review]
@yuexiao-wang thanks for this PR!
/approve
/lgtm

**Pull request merged**